src/sentry/utils/sdk.py

```diff
@@ -100,7 +101,8 @@ def capture_event(event):
         integrations=[
             DjangoIntegration(),
             CeleryIntegration(),
-            LoggingIntegration(event_level=None)
+            LoggingIntegration(event_level=None),
+            RustInfoIntegration(),
```

Comment:
Is this something that other customers could find useful as well?

Reply:
Yes but sadly it depends on our own exception types since there is no standard in python to carry foreign stacktraces ;(

Reply:
Boourns.